common/tools/dev-tool/src/commands/samples/checkNodeVersions.ts

```diff
@@ -224,7 +224,7 @@ export const commandInfo = makeCommandInfo(
     "node-versions": {
       kind: "string",
       description: "A comma separated list of node versions to use",
-      default: "10,12,14,15"
+      default: "12,14,15,16"
```

Comment:
Should we ditch 15 as well and add 17? 15 isn't active anymore, 17 is now current.

Reply:
Good point, done!